cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -51,7 +51,8 @@ func getResources(ctx context.Context, resourceType string, output string, args
 	if output != "" {
 		params = append(params, executables.WithOutput(output))
 	}
-	packages, err := kubectl.GetResources(ctx, resourceType, params...)
```

Comment:
Quick context: Initial implementation of GetResources was only needed for a single purpose. In order to refactor and reuse GetResources, it made sense to return the output of GetResources directly so that the user of the method is able to decide what to do with the result.